demisto_sdk/commands/format/format_module.py

```diff
@@ -122,6 +125,7 @@ def format_manager(
     interactive: bool = True,
     id_set_path: str = None,
     clear_cache: bool = False,
+    format_with_graph: bool = True
```

Comment:
Please rename to `use_graph` (no need to state `format_` in a command whose sole purpose is to format)